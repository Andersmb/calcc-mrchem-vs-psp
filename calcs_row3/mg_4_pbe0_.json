{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            0.0
          ],
          "radius": 1.72
        },
        {
          "center": [
            0.0,
            0.0,
            3.344815001
          ],
          "radius": 0.57
        },
        {
          "center": [
            0.0,
            0.0,
            -3.344815001
          ],
          "radius": 0.57
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "mg",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        },
        {
          "atom": "f",
          "xyz": [
            0.0,
            0.0,
            3.344815001
          ]
        },
        {
          "atom": "f",
          "xyz": [
            0.0,
            0.0,
            -3.344815001
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "mg_4_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        3.0851311328541564e-18
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.1069371727473856e-07,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            3.073856854360643e-09,
            -3.044214381641239e-08,
            1.0638104308569041e-07
          ],
          "vector_el": [
            2.9067992457498804e-09,
            -3.060926373477102e-08,
            1.0631431601737518e-07
          ],
          "vector_nuc": [
            1.6705760861076246e-10,
            1.6711991835863564e-10,
            6.672706831523101e-11
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Mg",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        },
        {
          "symbol": "F",
          "xyz": [
            0.0,
            0.0,
            3.344815001
          ]
        },
        {
          "symbol": "F",
          "xyz": [
            0.0,
            0.0,
            -3.344815001
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -46.94533896112544,
          -24.685722753456794,
          -24.68572265404424,
          -3.183101150423897,
          -1.9092227071044772,
          -1.909222705804168,
          -1.9017657101595962,
          -1.1183712758594224,
          -1.113069544759578,
          -0.4122655825191836,
          -0.39203536569431646,
          -0.37814495500526557,
          -0.3781448929486229,
          -0.3759252123463883,
          -0.3759251422177424,
          -46.94533896040982,
          -24.685722746042796,
          -24.68572265788198,
          -3.183101151797012,
          -1.909222712130899,
          -1.9092226989879726,
          -1.9017657177788103,
          -1.1183712751516786,
          -1.1130695437784217,
          -0.4122655846304717,
          -0.39203536563650776,
          -0.37814494234496765,
          -0.37814490545099727,
          -0.3759251965434522,
          -0.37592515779492053
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -219.52795722982978
      },
      "scf_energy": {
        "E_ee": 274.0963068477844,
        "E_eext": 0.0,
        "E_el": -476.40139755418295,
        "E_en": -1111.939834672507,
        "E_kin": 398.72563440427393,
        "E_next": 0.0,
        "E_nn": 76.68585554756066,
        "E_nuc": 76.68585554756066,
        "E_tot": -399.7155420066223,
        "E_x": -9.069423456105206,
        "E_xc": -28.21408067762904,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 277.54198400585864,
        "E_eext": 0.0,
        "E_el": -473.8968037279311,
        "E_en": -1111.6296855505711,
        "E_kin": 397.91707477156115,
        "E_next": 0.0,
        "E_nn": 76.68585554756066,
        "E_nuc": 76.68585554756066,
        "E_tot": -397.21094818037045,
        "E_x": -9.197319665226685,
        "E_xc": -28.528857289553038,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 270.0732233710649,
              "E_eext": 0.0,
              "E_el": -476.2255507875791,
              "E_en": -1100.7782959308909,
              "E_kin": 391.22744251805415,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.53969524001843,
              "E_x": -8.93985456822823,
              "E_xc": -27.808066177579125,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.53969524001843,
            "energy_update": 2.3287470596479807,
            "mo_residual": 0.5442665261671102,
            "wall_time": 165.945529451
          },
          {
            "energy_terms": {
              "E_ee": 275.16532203229224,
              "E_eext": 0.0,
              "E_el": -476.35909041808554,
              "E_en": -1111.8090620416988,
              "E_kin": 397.6912268636268,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.6732348705249,
              "E_x": -9.10508174591738,
              "E_xc": -28.301495526388337,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.6732348705249,
            "energy_update": 0.1335396305064478,
            "mo_residual": 0.29658797505544776,
            "wall_time": 120.825678582
          },
          {
            "energy_terms": {
              "E_ee": 272.6839828006213,
              "E_eext": 0.0,
              "E_el": -476.37688115607057,
              "E_en": -1108.814024653641,
              "E_kin": 396.8568765983727,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.6910256085099,
              "E_x": -9.024683415796952,
              "E_xc": -28.079032485626776,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.6910256085099,
            "energy_update": 0.017790737985023952,
            "mo_residual": 0.21164248645318848,
            "wall_time": 103.246309125
          },
          {
            "energy_terms": {
              "E_ee": 273.9044508682807,
              "E_eext": 0.0,
              "E_el": -476.3994099257163,
              "E_en": -1111.0811376411266,
              "E_kin": 398.0347524154706,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.71355437815566,
              "E_x": -9.06413155054552,
              "E_xc": -28.193344017795543,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.71355437815566,
            "energy_update": 0.022528769645759894,
            "mo_residual": 0.10096602608414261,
            "wall_time": 105.402141895
          },
          {
            "energy_terms": {
              "E_ee": 274.09728220888144,
              "E_eext": 0.0,
              "E_el": -476.4009036416019,
              "E_en": -1111.77839203051,
              "E_kin": 398.56391442894187,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.7150480940412,
              "E_x": -9.069967262254785,
              "E_xc": -28.213740986660287,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.7150480940412,
            "energy_update": 0.0014937158855445887,
            "mo_residual": 0.02207820287921512,
            "wall_time": 103.507291737
          },
          {
            "energy_terms": {
              "E_ee": 274.1121419413114,
              "E_eext": 0.0,
              "E_el": -476.4013550817739,
              "E_en": -1112.035176860246,
              "E_kin": 398.80831003092476,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.7154995342132,
              "E_x": -9.070135208790274,
              "E_xc": -28.2164949849737,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.7154995342132,
            "energy_update": 0.00045144017201437237,
            "mo_residual": 0.015521466186927357,
            "wall_time": 104.676265902
          },
          {
            "energy_terms": {
              "E_ee": 274.0950398023019,
              "E_eext": 0.0,
              "E_el": -476.401384025689,
              "E_en": -1111.9872553761093,
              "E_kin": 398.77481604145254,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.71552847812836,
              "E_x": -9.06946218516808,
              "E_xc": -28.214522308166202,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.71552847812836,
            "energy_update": 2.8943915140189347e-05,
            "mo_residual": 0.00389385167652116,
            "wall_time": 104.536343553
          },
          {
            "energy_terms": {
              "E_ee": 274.09169819672434,
              "E_eext": 0.0,
              "E_el": -476.4013948293988,
              "E_en": -1111.9355362653876,
              "E_kin": 398.72525770357186,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.7155392818381,
              "E_x": -9.069217598053394,
              "E_xc": -28.213596866254015,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.7155392818381,
            "energy_update": 1.0803709756146418e-05,
            "mo_residual": 0.00316580757163361,
            "wall_time": 109.206692544
          },
          {
            "energy_terms": {
              "E_ee": 274.0965403242845,
              "E_eext": 0.0,
              "E_el": -476.4013967773024,
              "E_en": -1111.9392724263707,
              "E_kin": 398.72471044828717,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.71554122974175,
              "E_x": -9.069369439679425,
              "E_xc": -28.21400568382395,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.71554122974175,
            "energy_update": 1.9479036268421623e-06,
            "mo_residual": 0.0009855404110758529,
            "wall_time": 107.579578917
          },
          {
            "energy_terms": {
              "E_ee": 274.09846869418845,
              "E_eext": 0.0,
              "E_el": -476.4013973357197,
              "E_en": -1111.9441177318306,
              "E_kin": 398.72795429259537,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.71554178815904,
              "E_x": -9.069460231258757,
              "E_xc": -28.21424235941414,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.71554178815904,
            "energy_update": 5.584172981798474e-07,
            "mo_residual": 0.0006666301351842479,
            "wall_time": 108.390050162
          },
          {
            "energy_terms": {
              "E_ee": 274.0968374962411,
              "E_eext": 0.0,
              "E_el": -476.40139751091647,
              "E_en": -1111.9398926741524,
              "E_kin": 398.7251908604318,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.7155419633558,
              "E_x": -9.069427638610584,
              "E_xc": -28.21410555482643,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.7155419633558,
            "energy_update": 1.7519676021038322e-07,
            "mo_residual": 0.00033642292153789586,
            "wall_time": 108.835897587
          },
          {
            "energy_terms": {
              "E_ee": 274.096042086902,
              "E_eext": 0.0,
              "E_el": -476.4013975441987,
              "E_en": -1111.9380381981302,
              "E_kin": 398.72404444540086,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.715541996638,
              "E_x": -9.069409300010795,
              "E_xc": -28.214036578360513,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.715541996638,
            "energy_update": 3.328221964693512e-08,
            "mo_residual": 0.00016574991834662626,
            "wall_time": 110.770510071
          },
          {
            "energy_terms": {
              "E_ee": 274.0963068477844,
              "E_eext": 0.0,
              "E_el": -476.40139755418295,
              "E_en": -1111.939834672507,
              "E_kin": 398.72563440427393,
              "E_next": 0.0,
              "E_nn": 76.68585554756066,
              "E_nuc": 76.68585554756066,
              "E_tot": -399.7155420066223,
              "E_x": -9.069423456105206,
              "E_xc": -28.21408067762904,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -399.7155420066223,
            "energy_update": 9.984262305806624e-09,
            "mo_residual": 0.0001027562768090865,
            "wall_time": 131.046686459
          }
        ],
        "wall_time": 1484.025704517
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
