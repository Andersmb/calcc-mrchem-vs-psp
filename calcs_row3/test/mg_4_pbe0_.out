                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Mg            :        0.000000        0.000000        0.000000
    1       F            :        0.000000        0.000000        3.344815
    2       F            :        0.000000        0.000000       -3.344815
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              15
 Beta electrons          :                                              15
 Total electrons         :                                              30
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999698651e-01
    1    1    a          :                              1.000000883212e+00
    2    1    a          :                              9.999996524782e-01
    3    1    a          :                              9.999998798574e-01
    4    1    a          :                              9.999990809500e-01
    5    1    a          :                              9.999998920668e-01
    6    1    a          :                              9.999998815500e-01
    7    1    a          :                              1.000023538321e+00
    8    1    a          :                              9.999799222854e-01
    9    1    a          :                              9.999980066193e-01
   10    1    a          :                              1.000006114655e+00
   11    1    a          :                              1.000015521392e+00
   12    1    a          :                              1.000016764050e+00
   13    1    a          :                              9.999901639904e-01
   14    1    a          :                              9.999800926870e-01
   15    1    b          :                              9.999999670397e-01
   16    1    b          :                              1.000000943956e+00
   17    1    b          :                              9.999993848772e-01
   18    1    b          :                              9.999998940872e-01
   19    1    b          :                              9.999990824934e-01
   20    1    b          :                              9.999998918600e-01
   21    1    b          :                              9.999998807280e-01
   22    1    b          :                              1.000017201672e+00
   23    1    b          :                              9.999799217174e-01
   24    1    b          :                              9.999938940393e-01
   25    1    b          :                              1.000002108361e+00
   26    1    b          :                              1.000016059505e+00
   27    1    b          :                              1.000016764050e+00
   28    1    b          :                              9.999897829901e-01
   29    1    b          :                              9.999800926912e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                397.917074771561
 E-N energy              :            (au)              -1111.629685550571
 Coulomb energy          :            (au)                277.541984005859
 Exchange energy         :            (au)                 -9.197319665227
 X-C energy              :            (au)                -28.528857289553
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 76.685855547561
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -473.896803727931
 Nuclear energy          :            (au)                 76.685855547561
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.972109481804e+02
                         :      (kcal/mol)             -2.492536311737e+05
                         :        (kJ/mol)             -1.042877193507e+06
                         :            (eV)             -1.080866005701e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -46.704346843532
    1    1    a          :            (au)                -24.156145204017
    2    1    a          :            (au)                -24.154993183577
    3    1    a          :            (au)                 -3.214938031203
    4    1    a          :            (au)                 -1.938061624370
    5    1    a          :            (au)                 -1.938051720691
    6    1    a          :            (au)                 -1.928598052954
    7    1    a          :            (au)                 -0.916961906685
    8    1    a          :            (au)                 -0.912240118565
    9    1    a          :            (au)                 -0.207059895885
   10    1    a          :            (au)                 -0.187197707764
   11    1    a          :            (au)                 -0.175535996110
   12    1    a          :            (au)                 -0.175052460641
   13    1    a          :            (au)                 -0.148530820337
   14    1    a          :            (au)                 -0.148440438590
   15    1    b          :            (au)                -46.704347061606
   16    1    b          :            (au)                -24.156136894194
   17    1    b          :            (au)                -24.154984983310
   18    1    b          :            (au)                 -3.214934260305
   19    1    b          :            (au)                 -1.938054609170
   20    1    b          :            (au)                 -1.938047936369
   21    1    b          :            (au)                 -1.928590859577
   22    1    b          :            (au)                 -0.916958561061
   23    1    b          :            (au)                 -0.912238182725
   24    1    b          :            (au)                 -0.207057965018
   25    1    b          :            (au)                 -0.187198640306
   26    1    b          :            (au)                 -0.175536273035
   27    1    b          :            (au)                 -0.175058269849
   28    1    b          :            (au)                 -0.148533862546
   29    1    b          :            (au)                 -0.148430570239
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -213.812262934233
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.477226e+00        -397.210948180370         -3.972109e+02
    1          5.442665e-01        -399.539695240018         -2.328747e+00
    2          2.965880e-01        -399.673234870525         -1.335396e-01
    3          2.116425e-01        -399.691025608510         -1.779074e-02
    4          1.009660e-01        -399.713554378156         -2.252877e-02
    5          2.207820e-02        -399.715048094041         -1.493716e-03
    6          1.552147e-02        -399.715499534213         -4.514402e-04
    7          3.893852e-03        -399.715528478128         -2.894392e-05
    8          3.165808e-03        -399.715539281838         -1.080371e-05
    9          9.855404e-04        -399.715541229742         -1.947904e-06
   10          6.666301e-04        -399.715541788159         -5.584173e-07
   11          3.364229e-04        -399.715541963356         -1.751968e-07
   12          1.657499e-04        -399.715541996638         -3.328222e-08
   13          1.027563e-04        -399.715542006622         -9.984262e-09
---------------------------------------------------------------------------
                      SCF converged in 13 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Mg            :        0.000000        0.000000        0.000000
    1       F            :        0.000000        0.000000        3.344815
    2       F            :        0.000000        0.000000       -3.344815
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                398.725634404274
 E-N energy              :            (au)              -1111.939834672507
 Coulomb energy          :            (au)                274.096306847784
 Exchange energy         :            (au)                 -9.069423456105
 X-C energy              :            (au)                -28.214080677629
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 76.685855547561
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -476.401397554183
 Nuclear energy          :            (au)                 76.685855547561
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.997155420066e+02
                         :      (kcal/mol)             -2.508252875156e+05
                         :        (kJ/mol)             -1.049453003646e+06
                         :            (eV)             -1.087681352401e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -46.945338961125
    1    1    a          :            (au)                -24.685722753457
    2    1    a          :            (au)                -24.685722654044
    3    1    a          :            (au)                 -3.183101150424
    4    1    a          :            (au)                 -1.909222707104
    5    1    a          :            (au)                 -1.909222705804
    6    1    a          :            (au)                 -1.901765710160
    7    1    a          :            (au)                 -1.118371275859
    8    1    a          :            (au)                 -1.113069544760
    9    1    a          :            (au)                 -0.412265582519
   10    1    a          :            (au)                 -0.392035365694
   11    1    a          :            (au)                 -0.378144955005
   12    1    a          :            (au)                 -0.378144892949
   13    1    a          :            (au)                 -0.375925212346
   14    1    a          :            (au)                 -0.375925142218
   15    1    b          :            (au)                -46.945338960410
   16    1    b          :            (au)                -24.685722746043
   17    1    b          :            (au)                -24.685722657882
   18    1    b          :            (au)                 -3.183101151797
   19    1    b          :            (au)                 -1.909222712131
   20    1    b          :            (au)                 -1.909222698988
   21    1    b          :            (au)                 -1.901765717779
   22    1    b          :            (au)                 -1.118371275152
   23    1    b          :            (au)                 -1.113069543778
   24    1    b          :            (au)                 -0.412265584630
   25    1    b          :            (au)                 -0.392035365637
   26    1    b          :            (au)                 -0.378144942345
   27    1    b          :            (au)                 -0.378144905451
   28    1    b          :            (au)                 -0.375925196543
   29    1    b          :            (au)                 -0.375925157795
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -219.527957229830
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000       -0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000       -0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 26m  6s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
